src/components/CardList/CardList.jsx

```diff
@@ -0,0 +1,27 @@
+import "./CardList.css";
+import Card from "./../UI/Card/Card";
+
+const CardList = (props) => {
+  //don't show cards where isDeleted property is true
+  const notDeletedCards = props.cardsText.filter((cardText) => {
+    return !cardText.isDeleted;
+  });
```

Comment:
Wrong. Once you actually remove elements from array - they will disappear from the component rendered list.
However the 'removal' logic is correct, you can use it in the parent component.